src/DropDown/DropDown.tsx

```diff
@@ -44,12 +44,15 @@ export const DropDown = (props: Props): JSX.Element => (
               {props.items.map((item: Item, index: number) => (
                 <StyledListItem
                   highlighted={index === highlightedIndex}
+                  selectedItem={item === selectedItem}
                   {...getItemProps({
                     item,
                     key: item.value,
                   })}
                 >
-                  {item.thumbnail && <StyledListItemImage alt={`${item.label} - thumbnail`} src={item.thumbnail} />}
+                  {selectedItem && item.thumbnail && (
```

Comment:
Is there a reason we are checking `selectedItem` and then `item.thumbnail` here? Are they meant to match?

Reply:
Good spot, I was testing something and forgot to take this out.